src/context.rs

```diff
@@ -6,27 +6,69 @@ use crate::dc_chat::*;
 use crate::dc_contact::*;
 use crate::dc_job::*;
 use crate::dc_jobthread::*;
-use crate::dc_log::*;
 use crate::dc_loginparam::*;
 use crate::dc_lot::dc_lot_t;
 use crate::dc_move::*;
 use crate::dc_msg::*;
 use crate::dc_receive_imf::*;
-use crate::dc_sqlite3::*;
 use crate::dc_stock::*;
 use crate::dc_tools::*;
 use crate::imap::*;
 use crate::key::*;
 use crate::smtp::*;
+use crate::sql::{self, Sql};
 use crate::types::*;
 use crate::x::*;
 
+const CONFIG_KEYS: [&'static str; 33] = [
```

Comment:
is there no way to avoid the "33" here?

Reply:
as long as it is an array, no

Reply:
this should become an enum eventually making this nicer, but one thing at a time *cough*

Reply:
fixed in the latest commit